### PR TITLE
Appendix C Subject Holder suggested editorial updates

### PR DESCRIPTION
**Pull request by grantnoble**
The sentences before and after the diagram seemed to be saying very much the same thing, so incorporated them into the opening para and deleted the second one.
Some wording changes, and some punctuation.
In "Issuer Authorizes Holder" section, the third para was an almost word for word duplication of the opening para. Deleted the third para.

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/695.html" title="Last updated on Jul 15, 2019, 7:44 PM UTC (8a38f4b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/695/96dc09e...grantnoble:8a38f4b.html" title="Last updated on Jul 15, 2019, 7:44 PM UTC (8a38f4b)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5502,20 +5499,19 @@ <h4>Subject Passes the Verifiable Credential to a Holder</h4>
 
         <p>
 Usually <a>verifiable credentials</a> are presented to <a>verifiers</a> by the
-<a>subject</a>. However in some cases, the <a>subject</a> might need to pass
-the whole or part of a <a>verifiable credential</a> to another <a>holder</a>.
-For example, if a patient (the <a>subject</a>) is too ill to take a
-prescription (the <a>verifiable credential</a>) to the pharmacist (the
-<a>verifier</a>), a friend might take the prescription in to pick up the
-medication.
+<a>subject</a>. However in some cases, the <a>subject</a> might need to pass the
```
Comment (TallTed) [inline index.html]: `However in` -> `However, in`

File: index.html [hunk 2]
```diff
@@ -5502,20 +5499,19 @@ <h4>Subject Passes the Verifiable Credential to a Holder</h4>
...
+whole or part of a <a>verifiable credential</a> to another <a>holder</a>. For
+example, if a patient (the <a>subject</a>) is too ill to take a prescription
+(the <a>verifiable credential</a>) to the pharmacist (the <a>verifier</a>), a
+friend might take the prescription in to pick up the medication.
         </p>
 
         <p>
-The data model allows for this, by the <a>subject</a> issuing a new
+The data model allows for this by the <a>subject</a> issuing a new
```
Comment (TallTed) [inline index.html]:
```
by the <a>subject</a> issuing a new
<a>verifiable credential</a> and giving
```
->
```
by letting the <a>subject</a> issue a new
<a>verifiable credential</a> and give
```

File: index.html (hunk 1 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: Done.

**Comment by grantnoble** [comment]
Updates completed.

**Comment by grantnoble** [comment]
Additional update completed.

**Pull request merged**